src/svd/device.py

```diff
@@ -1440,6 +1613,9 @@ def _extract_register_descriptions_helper(
         fields: Optional[Mapping[str, _FieldSpec]] = None
 
         if isinstance(element, bindings.RegisterElement):
+            if not FullRegisterProperties.is_full(reg_props):
+                raise ValueError(f"not full: {reg_props}")
```

Comment:
Super duper nit: Capitalization at start of string

Reply:
Updated this to raise a `SvdDefinitionError` with a more descriptive message.